app/serializers/api/v1/offers_package_serializer.rb

```diff
@@ -0,0 +1,22 @@
+module Api::V1
+  class OffersPackageSerializer < ApplicationSerializer
+    embed :ids, include: true
+    attributes :id, :offer_id, :item_id
```

Comment:
Should just be `package_id` and not `item_id`

Renaming on the backend for the sake of the Stock app cannot happen, ever.
Stock's mistakes should be handled in Stock, we don't bend the rules of the server for one app

:)